### PR TITLE
chats-list: UI refresh

### PR DESCRIPTION
**Pull request by GustavoNunes**
### Summary:
Android and iOS UI changes in the chats list screen.

What this contains:
- Fonts, colors, positioning, chat list padding, chat group icons.
- Small change to last message timestamp (added the time in "1 Sep 09:12")

What isn't covered (being done in other PRs):
- Toolbar style, actions (search, edit). Action button handler.
- Offline bar
- Other changes to last message timestamp (X minutes ago, add time to "yesterday")
- Tabs bar.

#### Notes

I don't like the way I implemented the chat list padding (Android-only): by using actual 8px views in the list header and footer. However, using regular CSS padding was not working and was taking me a lot of time, so I chose the easy way for now.

**status:** ready

### CODE REVIEW

**Comment by rasom** [comment]
screen shot 2017-02-27 at 15 36 19
1. Dapp badge has a wrong style
2. Shadow under bottom line

screen shot 2017-02-27 at 15 33 46

3. Separator should be 1pt thick

File: src/status_im/chats_list/screen.cljs [hunk 1]
```diff
@@ -63,24 +63,35 @@
...
    [list-view {:dataSource      (to-datasource chats)
                :renderRow       (fn [[id :as row] _ _]
                                   (list-item ^{:key id} [chat-list-item row]))
-               :renderFooter    #(list-item [chat-shadow-item])
-               :renderSeparator #(list-item
-                                   (when (< %2 (- (count chats) 1))
-                                     ^{:key (str "separator-" %2)}
-                                     [view st/chat-separator-wrapper
-                                      [view st/chat-separator-item]]))
+               :renderHeader    #(when-not (empty? chats) (list-item [chat-list-padding]))
```
Comment (rasom) [inline src/status_im/chats_list/screen.cljs]: https://github.com/bbatsov/clojure-style-guide#nil-punning

File: src/status_im/chats_list/screen.cljs [hunk 2]
```diff
@@ -63,24 +63,35 @@
...
-   [linear-gradient {:style  {:height 3}
+  [view {:height 12}
+   [chat-list-padding]
+   [linear-gradient {:style  {:height 4}
                      :colors st/gradient-top-bottom-shadow}]])
 
+(defn render-separator-fn [chats]
+  (fn [_ row-id _]
+    (list-item
+     (when (< row-id (- (count chats) 1))
```
Comment (rasom) [inline src/status_im/chats_list/screen.cljs]: `(dec (count chats))`

**Comment by flexsurfer** [comment]
image

**Comment by GustavoNunes** [comment]
@rasom, the changes requested are done.
I didn't fix the dapp badge style because @flexsurfer already did it in his PR.

About the shadow in iOS: I removed it, and noted that your reference image has a line after the last chat (bottom of the list), but the latest screen in Zeplin does not.

**Comment by GustavoNunes** [comment]
Hey @flexsurfer, I double checked, the separator color is gray, the same as specified in Zeplin: 
rgb 217 218 225 --> #d9dae1
About this, please also check [my comment at PR 828](https://github.com/status-im/status-react/pull/828#discussion_r103845300).

**Comment by GustavoNunes** [comment]
@rasom, should I rebase this branch against `0.9.4` or `epic/ui-refresh`?

**Comment by rasom** [comment]
@GustavoNunes, I'm going close all PRs into 0.9.4 branch, so you need to rebase your changes on `epic/ui-refresh` and file new PRs

**Pull request closed without merging**